modules/identity/src/Volo.Abp.PermissionManagement.Domain.Identity/Volo/Abp/PermissionManagement/Identity/RoleDeletedEventHandler.cs

```diff
@@ -1,26 +1,26 @@
 ﻿using System.Threading.Tasks;
 using Volo.Abp.Authorization.Permissions;
 using Volo.Abp.DependencyInjection;
-using Volo.Abp.Domain.Entities.Events;
+using Volo.Abp.Domain.Entities.Events.Distributed;
 using Volo.Abp.EventBus;
 using Volo.Abp.Identity;
 
 namespace Volo.Abp.PermissionManagement.Identity
 {
-    // public class RoleDeletedEventHandler :
-    //     ILocalEventHandler<EntityDeletedEventData<IdentityRole>>,
-    //     ITransientDependency
-    // {
-    //     protected IPermissionManager PermissionManager { get; }
-    //
-    //     public RoleDeletedEventHandler(IPermissionManager permissionManager)
-    //     {
-    //         PermissionManager = permissionManager;
-    //     }
-    //
-    //     public virtual async Task HandleEventAsync(EntityDeletedEventData<IdentityRole> eventData)
-    //     {
-    //         await PermissionManager.DeleteAsync(RolePermissionValueProvider.ProviderName, eventData.Entity.Name);
-    //     }
-    // }
+    public class RoleDeletedEventHandler :
+        ILocalEventHandler<EntityDeletedEto<IdentityRoleEto>>,
```

Comment:
I guess it should be `IDistributedEventHandler<EntityDeletedEto<IdentityRoleEto>>`

Reply:
Yes, it should be distributed. Thanks.